sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContainer.md

```diff
@@ -0,0 +1,203 @@
+applicable: SharePoint Online
+title: Get-SPOContainer
+schema: 2.0.0
+author: cindylay
+ms.author: cindylay
+ms.reviewer:
+---
+
+# Get-SPOContainer
+
+## SYNOPSIS
+
+Returns one or more Containers in a SharePoint repository services application. 
+
+## SYNTAX
+
+
+### ParamSet1
+
+```powershell
+Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [<CommonParameters>]
+```
+
+### ParamSet2
+```powershell
+Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [-Paged]
+```
+
+### ParamSet3
+```powershell
+Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [-Paged] [-PagingToken <Token String>]
+```
+
+### ParamSet4
+
+```powershell
+Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [[-Identity] <ContainerId>]
+```
+
+
+### ParamSet5
+
+```powershell
+Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [[-Identity] <ContainerSiteURL>]  
+```
+
+## DESCRIPTION
+
+The `Get-SPOContainer` cmdlet retrieves and returns a list of Containers and details of an individual Container created under a SharePoint repository services application. This command is available only in SharePoint Online Management Shell version 16.0.24211.12000 or higher to run this cmdlet.
+
+You need to be a SharePoint Online administrator or Global Administrator to run this cmdlet.
+
+> [!NOTE]  
+> Containers in the Recycle Bin will not be retrieved by using the `Get-SPOContainer` cmdlet. 
+
+
+> [!NOTE]
+> The OwningApplicationId for Microsoft Loop is `a187e399-0c36-4b98-8f04-1edc167a0996`.
+> 
+> The OwningApplicationId for Microsoft Designer is `5e2795e3-ce8c-4cfb-b302-35fe5cd01597`.
+ 
+ 
+
+## EXAMPLES
+
+### -----------------------EXAMPLE 1-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId 423poi45-jikl-9bnm-b302-1234ghy56789 | FT 
+```
+
+Example 1 returns a tabular list of Containers created under the SharePoint repository services application with the `OwningApplicationId` of  `423poi45-jikl-9bnm-b302-1234ghy56789`.
+
+To retrieve Containers for the Microsoft Loop app, use OwningApplicationId: `a187e399-0c36-4b98-8f04-1edc167a0996`. 
+
+To retrieve Containers for the Microsoft Designer app, use OwningApplicationId: `5e2795e3-ce8c-4cfb-b302-35fe5cd01597`.
+
+
+
+### -----------------------EXAMPLE 2-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId 423poi45-jikl-9bnm-b302-1234ghy56789 -Identity b66f5b2e-4cbd-4754-9ad3-8291c2c81ade 
+```
+
+Example 2 returns the detailed properties of the Container with associated `ContainerId`.
+
+ 
+### -----------------------EXAMPLE 3-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId 423poi45-jikl-9bnm-b302-1234ghy56789 -Identity https://contoso.sharepoint.com/storageContainers/CSP_b66f5b2e-4cbd-4754-9ad3-8291c2c81ade 
+```
+
+Example 3 gives the detailed properties of a container using site URL of a container.
+
+
+### -----------------------EXAMPLE 4-----------------------------
```

Comment:
```suggestion
### -----------------------EXAMPLE 4-----------------------------
```

Reply:
Please also similarly change all instances of the Example headings in the rest of the PR. Thanks.